atum/src/test/scala/za/co/absa/atum/BaseTestSuite.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum
+
+import za.co.absa.atum.model.{Checkpoint, ControlMeasure}
+
+object BaseTestSuite {
```

Comment:
how about making it a trait shared by tests named e.g. `ControlMeasureBaseTestSuite`?